src/main/java/com/sarality/db/UpdateOperation.java

```diff
@@ -0,0 +1,18 @@
+package com.sarality.db;
+
+/**
+ * Interface for defining updates to table schema from version to version
+ *
+ * @author Satya@ (Satya Puniani) on 01/03/17
+ */
+
+interface UpdateOperation {
```

Comment:
Can we call this something like SchemaUpdate and the Type as SchemaUpdateType.

As far as the interface goes, do we really need anything more that UpdateType and Db Version?